Buildings/Controls/OBC/FDE/PackagedRTUs/OperatingMode.mo

```diff
@@ -0,0 +1,264 @@
+    annotation (Placement(transformation(extent={{-112,-20},{-92,0}})));
+  Buildings.Controls.OBC.CDL.Continuous.Gain gai(
+    final k=TUpcntM)
+    annotation (Placement(transformation(extent={{-84,-20},{-64,0}})));
+  Buildings.Controls.OBC.CDL.Conversions.RealToInteger reaToInt
+    annotation (Placement(transformation(extent={{-56,-20},{-36,0}})));
+
+equation
+  connect(or2.u1,occ)  annotation (Line(points={{20,68},{14,68},{14,60},{-140,60}},
+                      color={255,0,255}));
+  connect(or2.y, yOcc)
+    annotation (Line(points={{44,68},{50,68},{50,54},{160,54}},
+                                               color={255,0,255}));
+  connect(not1.y, and2.u1) annotation (Line(points={{80,-10},{86,-10},{86,-28},{
+          92,-28}},  color={255,0,255}));
+  connect(not1.u, yOcc) annotation (Line(points={{56,-10},{50,-10},{50,54},{160,
+          54}},     color={255,0,255}));
+  connect(and2.y, ySBC) annotation (Line(points={{116,-28},{160,-28}},
+                           color={255,0,255}));
+  connect(and1.u1, and2.u1) annotation (Line(points={{92,-70},{86,-70},{86,-28},
+          {92,-28}},      color={255,0,255}));
+  connect(and1.y, ySBH)
+    annotation (Line(points={{116,-70},{160,-70}},color={255,0,255}));
+  connect(intGre.u1,occReq)
+    annotation (Line(points={{-16,30},{-140,30}}, color={255,127,0}));
+  connect(intGre.y, or2.u2) annotation (Line(points={{8,30},{14,30},{14,60},{20,
+          60}}, color={255,0,255}));
+  connect(and2.u2, intGre1.y)
+    annotation (Line(points={{92,-36},{52,-36},{52,-40},{10,-40}},
+                                                  color={255,0,255}));
+  connect(and1.u2, intGre2.y)
+    annotation (Line(points={{92,-78},{10,-78}},  color={255,0,255}));
+  connect(intGre1.u1,sbcReq)
+    annotation (Line(points={{-14,-40},{-140,-40}}, color={255,127,0}));
+  connect(intGre2.u1,sbhReq)
+    annotation (Line(points={{-14,-78},{-140,-78}}, color={255,127,0}));
+  connect(TotalTU, intToRea.u)
+    annotation (Line(points={{-140,-10},{-134,-10},{-134,-12},{-128,-12},{-128,-10},
+          {-114,-10}},                            color={255,127,0}));
+  connect(intToRea.y, gai.u)
+    annotation (Line(points={{-90,-10},{-86,-10}},
+                                                 color={0,0,127}));
+  connect(gai.y, reaToInt.u)
+    annotation (Line(points={{-62,-10},{-58,-10}},
+                                                 color={0,0,127}));
+  connect(intGre.u2, reaToInt.y) annotation (Line(points={{-16,22},{-28,22},{-28,
+          -10},{-34,-10}},
+                     color={255,127,0}));
+  connect(intGre1.u2, reaToInt.y) annotation (Line(points={{-14,-48},{-28,-48},{
+          -28,-10},{-34,-10}},
+                         color={255,127,0}));
+  connect(intGre2.u2, reaToInt.y) annotation (Line(points={{-14,-86},{-28,-86},{
+          -28,-10},{-34,-10}},
+                         color={255,127,0}));
+  annotation (defaultComponentName="OpMode",
```

Comment:
`opeMod`

Reply:
I'll look into this, but I think this is related to the issues we had trying to rename the validation model when the .mos file was in the wrong folder...